### DISCUSSION TITLE
Plotter.add_axes() with the "ipygany" backend

### DISCUSSION THREAD

**Discussion by tmatsuzawa** [pyvista/pyvista#1687] [Q&A] [answered]
The "ipygany" backend enables 3D interactive plotting on Jupyter notebook; however, some basic methods do not seem to work. Immediate ones I noticed are... Plotter.add_axes() and Plotter.add_bounding_box(). There must be more. 

1. How can I add axes?
2. How can I add a bounding box?

![image](https://user-images.githubusercontent.com/29457620/134444663-0cf3f8b5-9018-427b-ab1e-2b685e1a8d71.png)

Sample code:
```
import pyvista
pl = pyvista.Plotter()
actor = pl.add_mesh(pyvista.Box(), show_edges=True)
pl.add_axes()
pl.add_bounding_box()
pl.show(jupyter_backend='ipygany')
```

**Comment by banesullivan** [comment, marked as answer]
AFAIK, ipygany does not support axes. This isn't something we can address in PyVista

**Reply by akaszynski** [reply]
Yes, that's correct. Also, there's not been too much activity in the ipygany repository, but you can try raising the issue here:
https://github.com/QuantStack/ipygany/issues
